src/protocols/bdx/BDXMessageUtils.h

```diff
@@ -80,10 +88,11 @@ struct TransferInit
 
     // Range Control (required)
     bool mWideRange;       ///< Set true to indicate mStartOffset and mDefLen are 64-bit, else 32-bit.
-    uint64_t mStartOffset; ///< Proposed start offset of data. 0 for no offset.
+    uint64_t mStartOffset; ///< Proposed start offset of data. 0 for no offset. Will be truncated if mWideRange is false.
```

Comment:
Is it really truncated, or is it just that the range will be [0..UINT32_MAX] if mWideRange is false?

Reply:
I'll fix these comments